src/models/athb.js

```diff
@@ -0,0 +1,163 @@
+import { pmv_calculation } from "./pmv_ppd.js";
+
+/**
+ * Return the PMV value calculated with the Adaptive Thermal Heat Balance
+ * Framework [27]_. The adaptive thermal heat balance (ATHB) framework
```

Comment:
add a reference in references.md and link to it here

Reply:
added